### ISSUE TITLE
Notes and Notes_trees Table Clean up

### ISSUE THREAD

**Issue by Davidezrajay** [Greenstand/treetracker-database#18]
The Notes and Notes_trees Tables are obsolete. 

I believe these tables should be removed. 

They still contain old data.

**Comment by petersenpdx** [comment]
I agree. What is the difference between these "notes" and the one on the tree_table? I'd be happy to delete this for you - or maybe move the notes on here to the notes on the tree_tree table.

**Comment by Davidezrajay** [comment]
If it is not much work to move the old notes to the tree table, that would cool. Otherwise they are old.

**Comment by deepwinter** [comment]
That's correct.  Since 'trees' are not actually trees but each capture of a tree, the 'note' is associated with each entry in 'trees' and there is no need for a join table.
